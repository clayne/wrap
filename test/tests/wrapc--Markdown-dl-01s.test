wrapc | /dev/null | -u | md-dl-01s.xq | 0
