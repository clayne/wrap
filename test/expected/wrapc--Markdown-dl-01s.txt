  (:
   : Term 1
   : : This is the first sentence.  This is the second sentence.  This is the
   :   third sentence.  This is the fourth sentence.
   :
   : Not part of the list.
   :
   : Term 2
   : : This is the first sentence.  This is the second sentence.  This is the
   :   third sentence.  This is the fourth sentence.
   :)
